example_data/sample_bundle.json

```diff
@@ -528,21 +634,37 @@
       }
     },
     {
-      "fullUrl": "PlanDefinition/38018b47-b29a-8b06-daf6-6c5f2577bffa",
+      "fullUrl": "PlanDefinition/20RG-205G0047-plan-definition",
       "resource": {
         "resourceType": "PlanDefinition",
-        "id": "38018b47-b29a-8b06-daf6-6c5f2577bffa",
+        "id": "20RG-205G0047-plan-definition",
         "status": "active",
+        "identifier": [
+          {
+            "value": "20RG-205G0047-plan-definition"
+          }
+        ],
```

Comment:
do we get any benefit from specifying an identifier here?

Reply:
Yes, when I attempted to write the data directly to the FHIR server as explained earlier.

Reply:
- Should probably be a compound identifier made up of the specimen number and the reason for test code as those together are unique
- Probably no need to have the `-plan-definition` part, as this idenfier is only queried by a resource type (so only PlanDefinitions will be searched)